frontend/src/pages/comparisons/Comparison.tsx

```diff
@@ -54,7 +54,8 @@ const ComparisonPage = () => {
             generateInitial={true}
             getAlternatives={tutorialAlternatives}
             length={tutorialLength}
-            redirectTo={`${baseUrl}/personal/feedback`}
+            //redirectTo={`${baseUrl}/personal/feedback`}
+            redirectTo={`${baseUrl}/comparison`}
```

Comment:
We could add a `tutorialRedirectTo` key in the polls' configuration (in `utils/constants.tsx`) so that the page displayed after the tutorial is specific to each poll.